health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -364,239 +622,298 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+            });
+          }}
+          shapeOnOff={true}
+        />
+      </div>
+      {showAddPopup && (
+        <PopUp
+          className={"custom-pop-up-mapping"}
+          // className={"dataMapping"}
+          type={"default"}
+          heading={t("MAPPING_ADD_DATA")}
+          children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)